templates/deploy.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-deploy
```

Comment:
I think it is common in helm-charts to have the names be `{{ template "fullname" . }}` unless there is conflict.